backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -317,37 +317,33 @@ def lambda_handler(event, context):
                 if "x-mlspace-dataset-type" in event["headers"] and "x-mlspace-dataset-scope" in event["headers"]:
                     target_type = event["headers"]["x-mlspace-dataset-type"]
                     target_scope = event["headers"]["x-mlspace-dataset-scope"]
-                    if target_type == DatasetType.GLOBAL:
+                    if Permission.ADMIN in user.permissions:
```

Comment:
We will want to get Dustin's changes pulled in here so they are accounted for when checking permissions like this